configurable-rules/json-schema-misconfigurations/redocly.yaml

```diff
@@ -0,0 +1,42 @@
+rules:
+  rule/json-schema-string-misconfiguration:
+    subject: 
+      type: Schema
+    where: 
+      - subject: 
+          type: Schema
+          property: type
+        assertions:  
+          const: string
+    assertions: 
+      disallowed: 
+        - minimum
+        - maximum
+  rule/json-schema-array-misconfiguration:
+    subject: 
+      type: Schema
+      property: 
+        - properties
+    where: 
+      - subject: 
+          type: Schema
+          property: type
+        assertions:  
+          const: array
+    assertions: 
+      disallowed: 
+        - properties
+        - foo
```

Comment:
Why does it need to be disallowed?

Reply:
My bad:

![image](https://github.com/Redocly/redocly-cli-cookbook/assets/1161871/72369c4f-3904-41da-ab16-bab86d69f0fb)
 

Need to "save" before commit and push. 🤣